metric2/batch/batch.go

```diff
@@ -0,0 +1,11 @@
+package batch
```

Comment:
The other option would be to use `number.Number` instead of `Measurement`.

Reply:
It would be nice to have interfaces like this at the top-level _metric_ package.

```
package metric 

type Adder interface {
    Add(context.Context, number.Number, attrs ...attribute.KeyValue)
}

type Recorder interface {
    Record(context.Context, number.Number, attrs ...attribute.KeyValue)
}
```